docs/docs-beta/docs/guides/monitor-alert/alerting.md

```diff
@@ -1,12 +1,319 @@
+
+
+  </TabItem>
+  <TabItem value='pagerduty' label='PagerDuty'>
+    :::note
+You will need sufficient permissions in PagerDuty to add or edit services.
+:::
+
+In PagerDuty, you can either:
+
+- [Create a new service](https://support.pagerduty.com/main/docs/services-and-integrations#create-a-service), and add Dagster+ as an integration, or
+- [Edit an existing service](https://support.pagerduty.com/main/docs/services-and-integrations#edit-service-settings) to include Dagster+ as an integration
+
+When configuring the integration, choose **Dagster+** as the integration type, and choose an integration name in the format `dagster-plus-{your_service_name}`.
+
+After adding your new integration, you will be taken to a screen containing an **Integration Key**. This value will be used to configure your alerts.
+
+  </TabItem>
+  <TabItem value='microsoft_teams' label='Microsoft Teams'>
+    Create an incoming webhook by following the [Microsoft Teams documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=newteams%2Cdotnet).
+
+This will provide you with a webhook URL which can be used to configure your alerts.
+
+  </TabItem>
+</Tabs>
+
+## Alerting when a run fails
+
+You can set up alerts to notify you when a run fails.
+
+By default, these alerts will target all runs in the deployment, but they can be scoped to runs with a specific tag.
+        
+<Tabs groupId="ui_or_code">
+  <TabItem value='ui' label='In the UI'>
+    In the Dagster+ UI, navigate to **Deployment > Alerts**, click 'Add alert', and select `Run alert` from the dropdown menu.
+
+If desired, select a `target` from the dropdown menu to scope this alert to runs with a specific tag.
+
+  </TabItem>
+  <TabItem value='code' label='In code'>
+    
+
+  Execute the following command to sync the configured alert policy to your Dagster+ deployment.
+
+  ```bash
+  dagster-cloud deployment alert-policies sync -a /path/to/alert_policies.yaml
+  ```
+  <Tabs groupId="notification_service">
+    <TabItem value='email' label='Email'>
+      <CodeExample filePath="guides/monitor-alert/alerting/schedule-sensor-failure-email.yaml" language="yaml" />
+    </TabItem>
+    <TabItem value='slack' label='Slack'>
+      <CodeExample filePath="guides/monitor-alert/alerting/schedule-sensor-failure-slack.yaml" language="yaml" />
+    </TabItem>
+    <TabItem value='pagerduty' label='PagerDuty'>
+      <CodeExample filePath="guides/monitor-alert/alerting/schedule-sensor-failure-pagerduty.yaml" language="yaml" />
+    </TabItem>
+    <TabItem value='microsoft_teams' label='Microsoft Teams'>
+      <CodeExample filePath="guides/monitor-alert/alerting/schedule-sensor-failure-microsoft_teams.yaml" language="yaml" />
+    </TabItem>
+  </Tabs>
+
+  </TabItem>
+</Tabs>
+
+## Alerting when a run is taking a long time to complete
+
+You can set up alerts to notify you when a run is taking a long time to complete.
```

Comment:
We may also want to point out that users can set the threshold for how long is 'too long', otherwise it may be interpreted as 'Dagster+ will decide how long is too long and then alert me'